vital/tests/rpc_reader.h

```diff
@@ -79,6 +80,9 @@ kwiver::vital::simple_camera_rpc read_rpc( std::string filename )
           case 7:
             image_offset( word_idx ) = value;
             break;
+          case 8:
+            image_dimension( word_idx ) = static_cast<int>(value);
```

Comment:
Sorry to comment late but this is not going to be executed and is not needed unless an additional line is added to the `rpc_data.dat` file in the `data` directory. We probably should add data to `rpc_data.dat` and then expand the ```clone``` test in `test_camera_rpc.cxx` to check the dimensions.

Reply:
+1,  I assume we can make up any dimensions for those rpc files right?  Maybe some files should not have the dimensions and we should test that those read in a zeros.

Reply:
Yes - we can make up any dimensions. Right now for these tests I only have one set of rpc data but we could add more.

Reply:
Changes look good to me.